action.yml

```
@@ -0,0 +1,94 @@
+name: Sync Templates
+description: Synchronizes Templates between your repository and Fiberplane
+branding:
+  icon: tag
+  color: gray-dark
+
+inputs:
+  api-token:
+    description: API token used to access the Fiberplane API with
+    required: true
+  workspace-id:
+    description: ID of the workspace to which the templates should be uploaded to
+    required: true
+  fp-base-url:
+    description: Base URL of the Fiberplane API
+    default: https://studio.fiberplane.com
+  templates-directory:
+    description: "Custom directory that should be monitored for Template JSONNET files (default: .fiberplane/)"
```

The documented default does not match what is specified below

```suggestion
    description: "Custom directory that should be monitored for Template JSONNET files (default: .fiberplane/templates/)"
```